AppCenterAnalytics/AppCenterAnalytics/Internals/Util/MSAnalyticsConstants.h

```diff
@@ -7,3 +7,13 @@
 static const int kMSLongMetadataTypeId = 4;
 static const int kMSDoubleMetadataTypeId = 6;
 static const int kMSDateTimeMetadataTypeId = 9;
+
+/**
+ * Minimum flush interval for channel.
+ */
+static NSUInteger const MSFlushIntervalMinimum = 3.0;
+
+/**
+ * Maximum flush interval for channel.
+ */
+static NSUInteger const MSFlushIntervalMaximum = 24 * 60 * 60;
```

Comment:
Need the `k` prefix.
```suggestion
static NSUInteger const kMSFlushIntervalMaximum = 24 * 60 * 60;
```